superset/connectors/sqla/models.py

```diff
@@ -643,7 +635,10 @@ def get_sqla_query(  # sqla
 
         select_exprs += metrics_exprs
 
-        labels_expected = [str(c.name) for c in select_exprs]
+        labels_expected = None
+        if not db_engine_spec.supports_column_aliases or \
```

Comment:
I am wondering what would break if we always executed this codepath ? ie we always set labels_expected to be a valid array.

Reply:
I was thinking the exact same thing. I think it makes sense to always do it.